### DISCUSSION TITLE
Redirecting to actual definition

### DISCUSSION THREAD

**Discussion by Gerbuuun** [nuxtrdev/nuxtr-vscode#148] [Ideas]
I don't know if this is possible but redirecting a `cmd+click` to the actual definition instead of the generated type declaration file would be amazing.

Nuxt auto imports are great but sometimes I really wish to be able to "go to definition" to find a file.
I don't know if VSCode allows hooking into this feature to modify it but lets hope it can!

**Comment by oom-** [comment]
CTRL + P override with full name would be a + too

`Button > Text.vue => ButtonText` and not only `Text` (I suppose it's linked)
![image](https://github.com/user-attachments/assets/f7b84617-edc1-4163-a675-b195c29a97f7)
